### DISCUSSION TITLE
Problems editing landing page

### DISCUSSION THREAD

**Discussion by toohka42** [colinwilson/lotusdocs#139] [Q&A] [answered]
Hi, I have run into 2 problems editing the landing page.

I have copied the `landing.yaml` file from the [example site directory](https://github.com/colinwilson/lotusdocs/blob/release/exampleSite/data/landing.yaml) and started removing sections, as I only want to use the `image text` section template, but when running `hugo server`, I get the following error:

```
render: failed to render pages: render of "home" failed:
"/home/toohka/.cache/hugo_cache/modules/filecache/modules/pkg/mod/github.com/colinwilson/lotusdocs@v0.1.0/layouts/index.html:19:11":
execute of template failed at <partialCached $path ($indexContext.Get "indexContext")>:
error calling partialCached: partial "landing/imageText.html" not found 
```

Rendering fails on my linux workstation, on the GitLab pipeline and on the Cloudflare pages.

The problem seems to be in the `image text` section, because when I tried rendering template code from the [lotusdocs.dev](https://github.com/colinwilson/lotusdocs.dev/blob/release/data/landing.yaml) repository, which doesn't contain it, everything was fine.

The second problem I have is that when I tried removing all sections except the `hero` section, the rendered output still featured all template sections including `feature grid` and `image compare`.

I would appreciate any help.

**Comment by artberger** [comment, marked as answer]
Hi, I had opened an [issue](https://github.com/colinwilson/lotusdocs/issues/128) with some of these on the Landing Page. Colin mentioned that many of these will be resolved in the `0.2.0` release. For example, I don't think the `image text` was in `0.1.0` at all.

**Reply by toohka42** [reply]
Thanks, I have resolved it for now by forcing redirect to the docs page, I'll be looking forward to the `0.2.0` release.

**Reply by cspratik** [reply]
Hey, can you let me know how to force redirect docs page? Also, I want to host only /docs part on my website for now. @Okarisu

**Reply by toohka42** [reply]
Hi, since I am serving the docs via Cloudflare Pages, I have set up the redirect using Page rules - from URL https://docs.mydomain.com forward to https://docs.mydomain.com/docs. This is platform-specific solution and might not work on other content serving platforms, @cspratik
